frontend/src/metabase/parameters/utils/date-formatting.ts

```diff
@@ -126,16 +126,24 @@ function parseDateRangeValue(value: string) {
   return { start: moment(start, true), end: moment(end, true) };
 }
 
+function formatSingleDate(date: Moment) {
+  if (date.hours() || date.minutes()) {
+    return date.format("MMMM D, YYYY hh:mm a");
```

Comment:
```suggestion
    return date.format("MMMM D, YYYY hh:mm A");
```

It's weird that `generateTimeValueDescription` uses both [`a`](https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/frontend/src/metabase-lib/v1/queries/utils/query-time.js#L143) and [`A`](https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/frontend/src/metabase-lib/v1/queries/utils/query-time.js#L143).

I only found 2 usages of `a` in the project ([1](https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/frontend/src/metabase-lib/v1/queries/utils/query-time.js#L149), [2](https://github.com/metabase/metabase/blob/4e343c1da58cc0d460d3b367ac7343260bd3b888/frontend/src/metabase/lib/settings.ts#L252)). `A` is used everywhere else.

Let's make it `A` everywhere?